api/rpc_test.go

```diff
@@ -110,6 +115,188 @@ func TestModulesImplementFullAPI(t *testing.T) {
 	}
 }
 
+// TestAuthedRPC tests whether a client with admin permissions can access all
+// methods at all permission levels via RPC.
+func TestAuthedRPC(t *testing.T) {
```

Comment:
Can you rewrite the test cases using multiple `t.Run()`s? Think it would be more readable